SwiftUiMovieDb/Views/MoviePosterCard.swift

```diff
@@ -6,35 +6,33 @@
 //
 
 import SwiftUI
+import Kingfisher
 
 struct MoviePosterCard: View {
     
     let movie: Movie
-    
-    @ObservedObject var imageLoader = ImageLoader()
-    
+    let cache = ImageCache.default
     var body: some View {
         ZStack {
-            if self.imageLoader.image != nil {
-                Image(uiImage: self.imageLoader.image!)
-                    .resizable()
-                    .aspectRatio(contentMode: .fill)
-                    .cornerRadius(8)
-                    .shadow(radius: 4)
-            } else {
+            if cache.isCached(forKey: "my_cache_key") {
+              
                 Rectangle()
                     .fill(Color.gray.opacity(0.3))
                     .cornerRadius(8)
                     .shadow(radius: 4)
                 
                 Text(movie.title)
                 .multilineTextAlignment(.center)
+            } else {
+                KFImage(movie.posterURL)
+                    .cacheMemoryOnly()
+                    .resizable()
+                    .aspectRatio(contentMode: /*@START_MENU_TOKEN@*/.fill/*@END_MENU_TOKEN@*/)
```

Comment:
"@START_MENU_TOKEN" ? And use placeholder modifier on KFImage itself and not Rectangle ?